### DISCUSSION TITLE
add custom componet

### DISCUSSION THREAD

**Discussion by sarinaa-m** [alibaba/formily#3014] [Q&A 1.x] [answered]
Hi everyone,
I am using formily/react for creating form but I need to add more components to it. for example, I need to add a component but do not use antd Select component. 
How can I do that?

**Comment by janryWang** [comment, marked as answer]
https://github.com/alibaba/formily/tree/formily_next/packages/antd/src 

The components in this package are all custom components, you can imitate your own implementation
